config/settings.yml

```diff
@@ -1072,19 +1075,32 @@
   :custom_logo: false
   :events:
     :disk_usage_gt_percent: 80
+  :heartbeat_timeout: 2.minutes
+  :heartbeat_frequency: 30.seconds
```

Comment:
@yrudman this key is really confusing since we have heartbeat_freq in line 1194 👇 

@Fryguy is it safe to rename heartbeat_freq to heartbeat_frequency?  Otherwise, I'd suggest this one change to match the shortened version (yuck!)